qt5-5.11.2-2 package broken

After upgrading to `mingw-w64-x86_64-qt5-5.11.2-2`, GCC fails to find the Qt5 header files. Downgrading to `mingw-w64-x86_64-qt5-5.11.2-1` fixes the problem.

Here's an excerpt of our CI output:

```
[06:50:42][Step 3/5] In file included from K:/c401303cba1b4098/src/cli/Add.cpp:28:
[06:50:42][Step 3/5] K:/c401303cba1b4098/src/core/Entry.h:22:10: fatal error: QColor: No such file or directory
[06:50:42][Step 3/5]  #include <QColor>
[06:50:42][Step 3/5]           ^~~~~~~~
[06:50:42][Step 3/5] compilation terminated.
[06:50:42][Step 3/5] mingw32-make[2]: *** [src/cli/CMakeFiles/cli.dir/build.make:63: src/cli/CMakeFiles/cli.dir/Add.cpp.obj] Error 1
[06:50:42][Step 3/5] mingw32-make[2]: *** Waiting for unfinished jobs....
[06:50:42][Step 3/5] In file included from K:/c401303cba1b4098/src/cli/Clip.cpp:30:
[06:50:42][Step 3/5] K:/c401303cba1b4098/src/core/Entry.h:22:10: fatal error: QColor: No such file or directory
[06:50:42][Step 3/5]  #include <QColor>
[06:50:42][Step 3/5]           ^~~~~~~~
[06:50:42][Step 3/5] compilation terminated.
[06:50:42][Step 3/5] mingw32-make[2]: *** [src/cli/CMakeFiles/cli.dir/build.make:76: src/cli/CMakeFiles/cli.dir/Clip.cpp.obj] Error 1
[06:50:42][Step 3/5] mingw32-make[1]: *** [CMakeFiles/Makefile2:958: src/cli/CMakeFiles/cli.dir/all] Error 2
[06:50:42][Step 3/5] mingw32-make[1]: *** Waiting for unfinished jobs....
[06:50:43][Step 3/5] K:/c401303cba1b4098/src/core/Tools.cpp:24:10: fatal error: QImageReader: No such file or directory
[06:50:43][Step 3/5]  #include <QImageReader>
[06:50:43][Step 3/5]           ^~~~~~~~~~~~~~
[06:50:43][Step 3/5] compilation terminated.
[06:50:43][Step 3/5] [ 15%] Building CXX object src/CMakeFiles/autotype.dir/autotype/AutoType.cpp.obj
[06:50:43][Step 3/5] [ 15%] Building CXX object src/proxy/CMakeFiles/proxy.dir/__/browser/NativeMessagingBase.cpp.obj
[06:50:43][Step 3/5] mingw32-make[2]: *** [src/CMakeFiles/autotype.dir/build.make:63: src/CMakeFiles/autotype.dir/core/Tools.cpp.obj] Error 1
[06:50:43][Step 3/5] mingw32-make[2]: *** Waiting for unfinished jobs....
[06:50:43][Step 3/5] [ 15%] Building CXX object src/proxy/CMakeFiles/proxy.dir/NativeMessagingHost.cpp.obj
[06:50:44][Step 3/5] [ 15%] Building CXX object src/proxy/CMakeFiles/proxy.dir/proxy_autogen/mocs_compilation.cpp.obj
[06:50:45][Step 3/5] In file included from K:/c401303cba1b4098/src/autotype/AutoType.cpp:33:
[06:50:45][Step 3/5] K:/c401303cba1b4098/src/core/Entry.h:22:10: fatal error: QColor: No such file or directory
[06:50:45][Step 3/5]  #include <QColor>
[06:50:45][Step 3/5]           ^~~~~~~~
[06:50:45][Step 3/5] compilation terminated.
[06:50:45][Step 3/5] mingw32-make[2]: *** [src/CMakeFiles/autotype.dir/build.make:76: src/CMakeFiles/autotype.dir/autotype/AutoType.cpp.obj] Error 1
[06:50:45][Step 3/5] mingw32-make[1]: *** [CMakeFiles/Makefile2:147: src/CMakeFiles/autotype.dir/all] Error 

...

[06:50:48][Step 3/5] In file included from C:/msys64/mingw64/include/QtTest/qtest.h:349,
[06:50:48][Step 3/5]                  from C:/msys64/mingw64/include/QtTest/QtTest:11,
[06:50:48][Step 3/5]                  from K:/c401303cba1b4098/tests/modeltest.cpp:37:
[06:50:48][Step 3/5] C:/msys64/mingw64/include/QtTest/qtestsystem.h: In function 'bool QTest::qWaitForWindowActive(QWidget*, int)':
[06:50:48][Step 3/5] C:/msys64/mingw64/include/QtTest/qtestsystem.h:128:41: error: cannot convert 'QWindow*' to 'QWidget*'
[06:50:48][Step 3/5]              return qWaitForWindowActive(window, timeout);
[06:50:48][Step 3/5]                                          ^~~~~~
[06:50:48][Step 3/5] C:/msys64/mingw64/include/QtTest/qtestsystem.h:125:72: note:   initializing argument 1 of 'bool QTest::qWaitForWindowActive(QWidget*, int)'
[06:50:48][Step 3/5]      Q_REQUIRED_RESULT inline static bool qWaitForWindowActive(QWidget *widget, int timeout = 5000)
[06:50:48][Step 3/5]                                                                ~~~~~~~~~^~~~~~
[06:50:48][Step 3/5] In file included from C:/msys64/mingw64/include/QtWidgets/qwidget.h:44,
[06:50:48][Step 3/5]                  from C:/msys64/mingw64/include/QtWidgets/QWidget:1,
[06:50:48][Step 3/5]                  from C:/msys64/mingw64/include/QtTest/qtestsystem.h:50,
[06:50:48][Step 3/5]                  from C:/msys64/mingw64/include/QtTest/qtest.h:349,
[06:50:48][Step 3/5]                  from C:/msys64/mingw64/include/QtTest/QtTest:11,
[06:50:48][Step 3/5]                  from K:/c401303cba1b4098/tests/modeltest.cpp:37:
[06:50:48][Step 3/5] C:/msys64/mingw64/include/QtGui/qwindowdefs.h:54:7: note: class type 'QWindow' is incomplete
[06:50:48][Step 3/5]  class QWindow;
[06:50:48][Step 3/5]        ^~~~~~~
[06:50:48][Step 3/5] In file included from C:/msys64/mingw64/include/QtTest/qtest.h:349,
[06:50:48][Step 3/5]                  from C:/msys64/mingw64/include/QtTest/QtTest:11,
[06:50:48][Step 3/5]                  from K:/c401303cba1b4098/tests/modeltest.cpp:37:
[06:50:48][Step 3/5] C:/msys64/mingw64/include/QtTest/qtestsystem.h: In function 'bool QTest::qWaitForWindowExposed(QWidget*, int)':
[06:50:48][Step 3/5] C:/msys64/mingw64/include/QtTest/qtestsystem.h:135:42: error: cannot convert 'QWindow*' to 'QWidget*'
[06:50:48][Step 3/5]              return qWaitForWindowExposed(window, timeout);
[06:50:48][Step 3/5]                                           ^~~~~~
[06:50:48][Step 3/5] C:/msys64/mingw64/include/QtTest/qtestsystem.h:132:73: note:   initializing argument 1 of 'bool QTest::qWaitForWindowExposed(QWidget*, int)'
[06:50:48][Step 3/5]      Q_REQUIRED_RESULT inline static bool qWaitForWindowExposed(QWidget *widget, int timeout = 5000)
[06:50:48][Step 3/5]                                                                 ~~~~~~~~~^~~~~~
[06:50:48][Step 3/5] In file included from C:/msys64/mingw64/include/QtWidgets/qwidget.h:44,
[06:50:48][Step 3/5]                  from C:/msys64/mingw64/include/QtWidgets/QWidget:1,
[06:50:48][Step 3/5]                  from C:/msys64/mingw64/include/QtTest/qtestsystem.h:50,
[06:50:48][Step 3/5]                  from C:/msys64/mingw64/include/QtTest/qtest.h:349,
[06:50:48][Step 3/5]                  from C:/msys64/mingw64/include/QtTest/QtTest:11,
[06:50:48][Step 3/5]                  from K:/c401303cba1b4098/tests/modeltest.cpp:37:
[06:50:48][Step 3/5] C:/msys64/mingw64/include/QtGui/qwindowdefs.h:54:7: note: class type 'QWindow' is incomplete
[06:50:48][Step 3/5]  class QWindow;
[06:50:48][Step 3/5]        ^~~~~~~
```

--- a/mingw-w64-qt5-static/0042-qt-5.4.0-static-cmake-also-link-plugins-and-plugin-deps.patch
+++ b/mingw-w64-qt5-static/0042-qt-5.4.0-static-cmake-also-link-plugins-and-plugin-deps.patch
@@ -1,7 +1,7 @@
 diff -urN qt-everywhere-opensource-src-5.6.0.orig/qtbase/mkspecs/features/data/cmake/Qt5BasicConfig.cmake.in qt-everywhere-opensource-src-5.6.0/qtbase/mkspecs/features/data/cmake/Qt5BasicConfig.cmake.in
 --- qt-everywhere-opensource-src-5.6.0.orig/qtbase/mkspecs/features/data/cmake/Qt5BasicConfig.cmake.in	2015-08-26 15:31:26.194016900 +0100
 +++ qt-everywhere-opensource-src-5.6.0/qtbase/mkspecs/features/data/cmake/Qt5BasicConfig.cmake.in	2015-08-26 15:21:30.905016900 +0100
-@@ -58,14 +58,19 @@
+@@ -58,14 +58,24 @@
      set(imported_location \"$${CMAKE_DLL_DIR}${LIB_LOCATION}\")
  !!ENDIF
      _qt5_$${CMAKE_MODULE_NAME}_check_file_exists(${imported_location})
@@ -9,7 +9,7 @@ diff -urN qt-everywhere-opensource-src-5.6.0.orig/qtbase/mkspecs/features/data/c
 +        set(_list_sep \";\")
 +    endif()
 +    set_property(TARGET Qt5::$${CMAKE_MODULE_NAME} APPEND PROPERTY
-+        INTERFACE_LINK_LIBRARIES \"$<$<CONFIG:${Configuration}>:${_Qt5$${CMAKE_MODULE_NAME}_LIB_DEPENDENCIES}>\" \"$<$<CONFIG:${Configuration}>:${_Qt5$${CMAKE_MODULE_NAME}_STATIC_${Configuration}_LIB_DEPENDENCIES}>\"
++        INTERFACE_LINK_LIBRARIES \"${_Qt5$${CMAKE_MODULE_NAME}_LIB_DEPENDENCIES}\" \"$<$<CONFIG:${Configuration}>:${_Qt5$${CMAKE_MODULE_NAME}_STATIC_${Configuration}_LIB_DEPENDENCIES}>\"
 +    )
      set_target_properties(Qt5::$${CMAKE_MODULE_NAME} PROPERTIES
 -        \"INTERFACE_LINK_LIBRARIES\" \"${_Qt5$${CMAKE_MODULE_NAME}_LIB_DEPENDENCIES}${_Qt5$${CMAKE_MODULE_NAME}_STATIC_${Configuration}_LIB_DEPENDENCIES}\"
@@ -21,6 +21,11 @@ diff -urN qt-everywhere-opensource-src-5.6.0.orig/qtbase/mkspecs/features/data/c
 -        \"IMPORTED_LINK_INTERFACE_LIBRARIES_${Configuration}\" \"${_Qt5$${CMAKE_MODULE_NAME}_LIB_DEPENDENCIES}${_Qt5$${CMAKE_MODULE_NAME}_STATIC_${Configuration}_LIB_DEPENDENCIES}\"
 +        \"IMPORTED_LINK_INTERFACE_LIBRARIES_${Configuration}\" \"${_Qt5$${CMAKE_MODULE_NAME}_LIB_DEPENDENCIES}${_list_sep}${_Qt5$${CMAKE_MODULE_NAME}_STATIC_${Configuration}_LIB_DEPENDENCIES}\"
      )
++    # provide static plugins for MINSIZEREL and RELWITHDEBINFO too
++    set_target_properties(${_Qt5$${CMAKE_MODULE_NAME} PROPERTIES
++        MAP_IMPORTED_CONFIG_MINSIZEREL Release
++        MAP_IMPORTED_CONFIG_RELWITHDEBINFO Release
++    )
  
  !!IF !isEmpty(CMAKE_WINDOWS_BUILD)
 @@ -238,17 +241,31 @@

--- a/mingw-w64-qt5-static/PKGBUILD
+++ b/mingw-w64-qt5-static/PKGBUILD
@@ -101,7 +101,7 @@ _ver_base=5.11.2
 # use 5.6.1-1 hot fix : only the archive name was changed from *-5.6.1.tar.xz to *-5.6.1-1.tar.xz
 _hotfix=
 pkgver=${_ver_base//-/}
-pkgrel=2
+pkgrel=3
 arch=('any')
 pkgdesc="A cross-platform application and UI framework (mingw-w64${_namesuff})"
 url='https://www.qt.io/'
@@ -740,7 +740,7 @@ sha256sums=('c6104b840b6caee596fa9a35bc5f57f67ed5a99d6a36497b6fe66f990a53ca81'
             '3ae9d64fb683036b3b1f91036378791e1ec4a10205c59e78761d22d8479c0540'
             'd720ac76035d56caea1f045dfb468a25865f9d4b22016cec5414f1c639a7697a'
             '914051840e01bf453e3db710d4471e6758e39e2cb87f8ffe3072fb90218c9f83'
-            'dbad9a04b7cd9ac7767a388e40d22141cfbf0c461c66e252254706a0af0b96b1'
+            '2ed8fa715d7459df66bdbb938d098df8ae64209b5ba65957d754eeef09c8f995'
             '29ed4566d4c1853b70a5173c6391ed90f123c5121b5836f2d16f8478f6354f0a'
             '7d7a99ff45c6ae4f39be663d52bafc125970787f364e84593117e6a7d79f7eae'
             '72e46178f2f694d8408f22684d1a7d39394056cb574fc4855c1f38c9d51e7e6c'

--- a/mingw-w64-qt5/0042-qt-5.4.0-static-cmake-also-link-plugins-and-plugin-deps.patch
+++ b/mingw-w64-qt5/0042-qt-5.4.0-static-cmake-also-link-plugins-and-plugin-deps.patch
@@ -1,7 +1,7 @@
 diff -urN qt-everywhere-opensource-src-5.6.0.orig/qtbase/mkspecs/features/data/cmake/Qt5BasicConfig.cmake.in qt-everywhere-opensource-src-5.6.0/qtbase/mkspecs/features/data/cmake/Qt5BasicConfig.cmake.in
 --- qt-everywhere-opensource-src-5.6.0.orig/qtbase/mkspecs/features/data/cmake/Qt5BasicConfig.cmake.in	2015-08-26 15:31:26.194016900 +0100
 +++ qt-everywhere-opensource-src-5.6.0/qtbase/mkspecs/features/data/cmake/Qt5BasicConfig.cmake.in	2015-08-26 15:21:30.905016900 +0100
-@@ -58,14 +58,19 @@
+@@ -58,14 +58,24 @@
      set(imported_location \"$${CMAKE_DLL_DIR}${LIB_LOCATION}\")
  !!ENDIF
      _qt5_$${CMAKE_MODULE_NAME}_check_file_exists(${imported_location})
@@ -9,7 +9,7 @@ diff -urN qt-everywhere-opensource-src-5.6.0.orig/qtbase/mkspecs/features/data/c
 +        set(_list_sep \";\")
 +    endif()
 +    set_property(TARGET Qt5::$${CMAKE_MODULE_NAME} APPEND PROPERTY
-+        INTERFACE_LINK_LIBRARIES \"$<$<CONFIG:${Configuration}>:${_Qt5$${CMAKE_MODULE_NAME}_LIB_DEPENDENCIES}>\" \"$<$<CONFIG:${Configuration}>:${_Qt5$${CMAKE_MODULE_NAME}_STATIC_${Configuration}_LIB_DEPENDENCIES}>\"
++        INTERFACE_LINK_LIBRARIES \"${_Qt5$${CMAKE_MODULE_NAME}_LIB_DEPENDENCIES}\" \"$<$<CONFIG:${Configuration}>:${_Qt5$${CMAKE_MODULE_NAME}_STATIC_${Configuration}_LIB_DEPENDENCIES}>\"
 +    )
      set_target_properties(Qt5::$${CMAKE_MODULE_NAME} PROPERTIES
 -        \"INTERFACE_LINK_LIBRARIES\" \"${_Qt5$${CMAKE_MODULE_NAME}_LIB_DEPENDENCIES}${_Qt5$${CMAKE_MODULE_NAME}_STATIC_${Configuration}_LIB_DEPENDENCIES}\"
@@ -21,6 +21,11 @@ diff -urN qt-everywhere-opensource-src-5.6.0.orig/qtbase/mkspecs/features/data/c
 -        \"IMPORTED_LINK_INTERFACE_LIBRARIES_${Configuration}\" \"${_Qt5$${CMAKE_MODULE_NAME}_LIB_DEPENDENCIES}${_Qt5$${CMAKE_MODULE_NAME}_STATIC_${Configuration}_LIB_DEPENDENCIES}\"
 +        \"IMPORTED_LINK_INTERFACE_LIBRARIES_${Configuration}\" \"${_Qt5$${CMAKE_MODULE_NAME}_LIB_DEPENDENCIES}${_list_sep}${_Qt5$${CMAKE_MODULE_NAME}_STATIC_${Configuration}_LIB_DEPENDENCIES}\"
      )
++    # provide static plugins for MINSIZEREL and RELWITHDEBINFO too
++    set_target_properties(${_Qt5$${CMAKE_MODULE_NAME} PROPERTIES
++        MAP_IMPORTED_CONFIG_MINSIZEREL Release
++        MAP_IMPORTED_CONFIG_RELWITHDEBINFO Release
++    )
  
  !!IF !isEmpty(CMAKE_WINDOWS_BUILD)
 @@ -238,17 +241,31 @@

--- a/mingw-w64-qt5/PKGBUILD
+++ b/mingw-w64-qt5/PKGBUILD
@@ -101,7 +101,7 @@ _ver_base=5.11.2
 # use 5.6.1-1 hot fix : only the archive name was changed from *-5.6.1.tar.xz to *-5.6.1-1.tar.xz
 _hotfix=
 pkgver=${_ver_base//-/}
-pkgrel=2
+pkgrel=3
 arch=('any')
 pkgdesc="A cross-platform application and UI framework (mingw-w64${_namesuff})"
 url='https://www.qt.io/'
@@ -740,7 +740,7 @@ sha256sums=('c6104b840b6caee596fa9a35bc5f57f67ed5a99d6a36497b6fe66f990a53ca81'
             '3ae9d64fb683036b3b1f91036378791e1ec4a10205c59e78761d22d8479c0540'
             'd720ac76035d56caea1f045dfb468a25865f9d4b22016cec5414f1c639a7697a'
             '914051840e01bf453e3db710d4471e6758e39e2cb87f8ffe3072fb90218c9f83'
-            'dbad9a04b7cd9ac7767a388e40d22141cfbf0c461c66e252254706a0af0b96b1'
+            '2ed8fa715d7459df66bdbb938d098df8ae64209b5ba65957d754eeef09c8f995'
             '29ed4566d4c1853b70a5173c6391ed90f123c5121b5836f2d16f8478f6354f0a'
             '7d7a99ff45c6ae4f39be663d52bafc125970787f364e84593117e6a7d79f7eae'
             '72e46178f2f694d8408f22684d1a7d39394056cb574fc4855c1f38c9d51e7e6c'

